presto-main/src/main/java/com/facebook/presto/sql/planner/SubqueryPlanner.java

```diff
@@ -452,30 +451,19 @@ private PlanBuilder appendApplyNode(
         return Optional.empty();
     }
 
-    private PlanBuilder createPlanBuilder(Expression expression)
+    private PlanBuilder createPlanBuilder(Node node)
```

Comment:
I can do that with subsequent PR, this PR get huge as for such simple fix I don't want to make even larger